CMakeLists.txt

```diff
@@ -1,8 +1,38 @@
 cmake_minimum_required(VERSION 3.14)
-project(notcurses VERSION 1.4.3
+
+# Shared library versioning is done in the same way as SDL:
+#
+# If the version changes, then:
+#
+#   NOTCURSES_VERSION_MICRO += 1;
+#   NOTCURSES_INTERFACE_AGE += 1;
+#   NOTCURSES_BINARY_AGE += 1;
+# if any functions/classes/methods have been added, set NOTCURSES_INTERFACE_AGE to 0.
+# if backwards compatibility has been broken,
+# set NOTCURSES_BINARY_AGE and NOTCURSES_INTERFACE_AGE to 0.
+
+set(NOTCURSES_VERSION_MAJOR 1)
+set(NOTCURSES_VERSION_MINOR 5)
```

Comment:
I'm gonna go ahead and set these to 1.4.5, and introduce this ASAP -- do you see any reason not to do this?

Reply:
Nope, sounds good :+1: :)